client.lua

```diff
@@ -1,25 +1,39 @@
 CreateThread(function()
+	local playerPed = PlayerPedId()
+	local boneId = 31086
+
 	RequestStreamedTextureDict("uhhh")
+
+	while not HasStreamedTextureDictLoaded("uhh")
+		Wait(0)
+	end
 	
-	while true do Wait(0)
-		local ratio = GetAspectRatio()
-		
+	while true do
 		local pool = GetGamePool("CPed")
-		table.sort(pool, function(a, b) return #(GetEntityCoords(a) - GetFinalRenderedCamCoord()) > #(GetEntityCoords(b) - GetFinalRenderedCamCoord()) end)
+		local camCoord = GetFinalRenderedCamCoord()
+
+		table.sort(pool, function(a, b) return #(GetEntityCoords(a) - camCoord) > #(GetEntityCoords(b) - camCoord) end)
 		
 		for i,v in pairs(pool) do
-			local boneId = 31086
+			playerPed = PlayerPedId()

```

Comment:
you update it afterall, so - why bother?

Reply:
imo its still better to make that variable outside of while loop, also i fucked up and put that insidie for loop, fixed